gormtestsupport/db_test_suite.go

```diff
@@ -85,7 +87,9 @@ func (s *DBTestSuite) PopulateDBTestSuite(ctx context.Context) {
 
 // TearDownSuite implements suite.TearDownAllSuite
 func (s *DBTestSuite) TearDownSuite() {
-	s.cleanSuite()
+	if s.Configuration.IsCleanTestDataEnabled() {
```

Comment:
Could you add a comment why this is being done?

Reply:
well, I thought that the comment on the `configuration.ConfigurationData.IsCleanTestDataEnabled()` was clear enough?

Reply:
Sure, but a line of comment explaining why would we have to sometimes disable the test data cleanup, would be helpful.

Reply:
ok, let me add this, then ;)